File: scholar-business/src/main/java/com/varun/scholar/business/data/entities1/ScheduleSearch.java
package com.varun.scholar.business.data.entities1;

import java.util.Date;

import javax.persistence.Entity;

import com.varun.scholar.business.data.entities.AbstractEntity;

@Entity
public class ScheduleSearch extends AbstractEntity {
	String courseName;
	String batchName;
	String subjectName;
	Date examDate;
}

File: scholar-web/src/main/webapp/app/model/staff/examination/schedule/ScheduleSearch.js
Ext.define('scholar.model.staff.examination.schedule.ScheduleSearch', {
	extend: 'Ext.data.Model',		
	fields : [ {
		name : 'id',
		type : 'int'
	}, {
			name : 'courseName',
			type : 'string'
		}, {
			name : 'batchName',
			type : 'string'
		},
		{
			name : 'subjectName',
			type : 'string'
		},
		{
			name : 'examDate',
			type : 'date',
			dateFormat : 'n/j h:ia'
		} ]
});